interp/interp_test.go

```
@@ -1526,6 +1549,41 @@ func TestInterpreter_Run(t *testing.T) {
+	} {
+		t.Run("ref set and get round-trip "+tt.name, func(t *testing.T) {
+			prog := program.New([]instr.Instruction{
+				instr.New(instr.GLOBAL_GET, 0),
+				instr.New(instr.REF_NEW),
+				instr.New(instr.DUP),
+				instr.New(instr.GLOBAL_GET, 1),
+				instr.New(instr.REF_SET),
+				instr.New(instr.REF_GET),
+			}, program.WithGlobals(tt.typ, tt.typ))
+			i := New(prog, WithThreshold(-1))
+			defer i.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `i.Close` is not checked

(errcheck)